PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -55,7 +55,7 @@
   </change-notes>
 
   <!-- please see https://confluence.jetbrains.com/display/IDEADEV/Build+Number+Ranges for description -->
-  <idea-version since-build="182" until-build="182.*"/>
+  <idea-version since-build="192" until-build="192.*"/>
```

Comment:
Maybe it's not a good practice to mix irrelevant version change thing in a bug fix PR.

Reply:
It only affects the IntelliJ language service, anyway, I will backout it.